src/js/bootstrap-slider.js

```diff
@@ -791,6 +791,11 @@ const windowIsDefined = (typeof window === "object");
 			// Bind window handlers
 			this.resize = this._resize.bind(this);
 			window.addEventListener("resize", this.resize, false);
+			// Simulate Popper's behavior of listening to both resize and scroll:
+			// https://popper.js.org/docs/v2/modifiers/event-listeners/
+			if (this.options.preventOverflow) {
+				window.addEventListener("scroll", this.resize, false);
```

Comment:
could you make sure this event listener is being removed in the `_removeSliderEventHandlers` cleanup method?